packaging/deb/polycli/DEBIAN/README.md

```diff
@@ -0,0 +1,20 @@
+# Debian
+
+
+
+For debian packages you will need to add the following layouts during the build
+
+
+
+polycli/
+   DEBIAN/control
+   usr/local/bin/polycli
```

Comment:
maybe `/usr/bin/polycli`?

Reply:
![Screenshot 2024-01-23 at 5 56 31 PM](https://github.com/maticnetwork/polygon-cli/assets/11021924/1ffade97-cc9d-4459-85ef-199194b2ddb6)

tested this out. i think this needs extra configs? i'll just leave it as-is for now